browser/ui/BUILD.gn

```diff
@@ -1166,6 +1166,14 @@ source_set("ui") {
       "//brave/components/brave_player/common",
     ]
   }
+
+  sources += brave_browser_ui_bookmark_sources
+  sources += brave_browser_ui_sidebar_sources
+  sources += brave_browser_ui_tabs_sources
```

Comment:
we only use these source/deps lists when we need to inject sources into chromium target.

here we're working with our `BUILD.gn`, so we should add those sources/deps directly into `//browser/ui/BUILD.gn`.

Reply:
Aha, thanks! Will update.